src/SectionsNavigation/SectionStackNavigator.cs

```diff
@@ -22,12 +22,14 @@ public class SectionStackNavigator : ISectionStackNavigator, IModalStackNavigato
 		/// <param name="name">The name of this section.</param>
 		/// <param name="isModal">Flag indicating whether this section is a modal.</param>
 		/// <param name="priority">The priority of the section.</param>
-		public SectionStackNavigator(IStackNavigator inner, string name, bool isModal, int priority)
+		/// <param name="closeModalTransitionInfo">The default transition info for the close modal operation. This is only relevant for modals.</param>
+		public SectionStackNavigator(IStackNavigator inner, string name, bool isModal, int priority, SectionsTransitionInfo closeModalTransitionInfo = null)
 		{
 			_inner = inner;
 			Name = name;
 			IsModal = isModal;
 			Priority = priority;
+			CloseModalTransitionInfo = closeModalTransitionInfo;
```

Comment:
Won't this throw if `closeModalTransitionInfo `is null ? `SectionsTransitionInfo CloseModalTransitionInfo `doesn't seem to be nullable

Reply:
The nullable feature is not enabled. It's expected to be null in all cases except one.